.github/workflows/pr.yml

```
@@ -52,6 +55,13 @@ jobs:
     if: ${{ needs.modified-files.outputs.app-changes == 'true' || needs.modified-files.outputs.e2e-changes == 'true' || needs.modified-files.outputs.fiftyone-changes == 'true' }}
     uses: ./.github/workflows/e2e.yml
 
+  frozen-docs:
```

actionlint (1.7.4)

"steps" section is missing in job "frozen-docs"

(syntax-check)